engine/runtime/src/main/java/org/enso/interpreter/runtime/EnsoContext.java

```diff
@@ -621,7 +623,7 @@ static Object lookupJavaClass(
   public TruffleObject lookupJavaClass(String className) {
     var collectedExceptions = new ArrayList<Exception>();
 
-    if (true) { // set this to false to simulate classloading via Channel
+    if (isHostClassLoading) {
```

Comment:
How to debug the dual mode via `Channel`? Run in the same JVM!
```
sbt:enso> runEngineDistribution --jvm -vm.D=polyglot.enso.hostClassLoading=false --run test/Generic_JDBC_Tests --debug
```
will load all classes _"like in the dual"_ mode while running in a normal HotSpot JVM.